.github/terraform/main.tf

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
This is such a great idea. I know we have an internal doc with what infra we need to recreate the CICD environment for Online Boutique, but it's nice to have it all codify and easily-reproducible in a TF script. Thanks!